Sources/MapboxMaps/Foundation/Camera/CameraAnimationsManager.swift

```diff
@@ -281,4 +283,39 @@ public class CameraAnimationsManager: CameraAnimationsManagerProtocol {
         cameraViewContainerView.addSubview(cameraView)
         return cameraView
     }
+
+    /// This function will handle the natural decelration of a gesture when there is a velocity provided. A use case for this is the pan gesture.
+    /// - Parameters:
+    ///   - location: Current location of center coordinate
+    ///   - velocity: The speed at which the map should move over time
+    ///   - decelerationRate: A multiplication factor that determines the speed at which the velocity should slow down
+    ///   - locationChangeHandler: Change handler to be passed through to the animator
+    ///   - completion: Completion to be called after animation has finished
+    internal func decelerate(location: CGPoint,
+                             velocity: CGPoint,
```

Comment:
Would `CGVector` be a better data type to represent velocity?

Reply:
Perhaps, but this is the type that UIGestureRecognizer uses, so it avoids any kind of conversion.

Reply:
ooh that's strange.